src/main/java/uk/gov/hmcts/ccd/endpoint/exceptions/CaseValidationException.java

```diff
@@ -2,11 +2,36 @@
 
 import org.springframework.http.HttpStatus;
 import org.springframework.web.bind.annotation.ResponseStatus;
+import uk.gov.hmcts.ccd.domain.model.std.CaseFieldValidationError;
+import uk.gov.hmcts.ccd.domain.model.std.CaseValidationError;
+
+import java.util.List;
 
 @ResponseStatus(code = HttpStatus.UNPROCESSABLE_ENTITY)
 public class CaseValidationException extends ValidationException {
 
-    public CaseValidationException() {
+    private final String[] fields;
+
+    public CaseValidationException(List<CaseFieldValidationError> fieldErrors) {
         super("Case data validation failed");
+
+        String[] fieldIds = new String[0];
```

Comment:
I'd recommend to prefer List than Array in Java. The perf gain of using Array is almost 0 but the boilerplate in the code is much higher because working with arrays in Java is very cumbersome